cookienotice/cookienotice.php

```diff
@@ -5,87 +5,66 @@
  * Description: Configure, show and handle a simple cookie notice
  * Version: 1.0
  * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
- * 
  */
-use Friendica\Core\Hook;
+
+use Friendica\App;
 use Friendica\Core\Config;
+use Friendica\Core\Hook;
 use Friendica\Core\L10n;
 use Friendica\Core\Renderer;
 
 /**
  * cookienotice_install
  * registers hooks
- * 
+ *
  * @return void
  */
 function cookienotice_install()
 {
 	Hook::register('page_content_top', __FILE__, 'cookienotice_page_content_top');
 	Hook::register('page_end', __FILE__, 'cookienotice_page_end');
-	Hook::register('addon_settings', __FILE__, 'cookienotice_addon_settings');
-	Hook::register('addon_settings_post', __FILE__, 'cookienotice_addon_settings_post');
-}
-
-/**
- * cookienotice_uninstall
- * unregisters hooks
- * 
- * @return void
-*/
-function cookienotice_uninstall()
```

Comment:
Why do you remove the `uninstall` routine?

Reply:
Definitely, but the documentation is in the core repo, nothing I can do here.

Reply:
Done: https://github.com/friendica/friendica/pull/7122

Reply:
Thanks